.github/workflows/CI.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: Trass3r/setup-cpp@master
+    - name: build
+      run: |
+        set -eux
+        sudo apt update && sudo apt -y upgrade
+        sudo apt install -y build-essential g++-mingw-w64-i686 libpng16-16
+        make DEBUG=1 GIT_VERSION=.$(git describe --always) PACKAGE_SUFFIX=tryme -j`nproc` -k heavylog
```

Comment:
Note, that I place the "." here on this line (and the default for `GIT_VERSION` below is completely blank). 
This means when KFX is built normally, it will have no trace of this extra version number. And when github runs this line, it will add e.g. ".a1b2c34" to the end of the version string (`VER_STRING`)